app/controllers/actions/resources/IntakeStage1Action.scala

```diff
@@ -3,19 +3,17 @@ package actions
 package resources
 
 import forms._
-
 import models.Truthy
 import util.IpmiCommand
 import util.concurrent.BackgroundProcessor
 import util.plugins.{IpmiPowerCommand, PowerManagement}
 import util.security.SecuritySpecification
-
 import collins.power.Identify
 import collins.power.management.{PowerManagement, PowerManagementConfig}
-
 import play.api.data.Form
 import play.api.data.Forms._
 import play.api.mvc.AsyncResult
+import com.twitter.util.Await
```

Comment:
Twitter. Collins has used twitter finagle internally, we should probably remove it and just use akka but that is out of scope for this particular PR